Project.toml

```diff
@@ -41,7 +41,7 @@ OrdinaryDiffEq = "5, 6"
 Plots = "1.39.0"
 SciMLBase = "1"
 StaticArrays = "1"
-SurfaceFluxes = "0.7"
+SurfaceFluxes = "0.8.0"
```

Comment:
we should be able to use "0.8", unless we expect not to be compatible with versions 0.8.1 or greater